httpserver/apis/v1/apis.go

```diff
@@ -31,3 +31,15 @@ const (
 	// ReadyScanResponseType indicates that a server has successfully completed a request
 	ReadyScanResponseType ScanResponseType = "ready"
 )
+
+// A WorkloadScan contains the identifiers of a workload to be scanned
```

Comment:
I think we should use an `IBasic` compatible object, this way we won't need to convert between objects.

Reply:
As discussed - I made it compatible with `IMetadata`